src/main/java/seedu/address/model/person/Person.java

```diff
@@ -89,17 +119,27 @@ public boolean equals(Object other) {
         }
 
         Person otherPerson = (Person) other;
+
+        //Verify visit todos separately because .equals doesn't work with Collection<>
+        Iterator otherPersonVisitTodos = otherPerson.getVisitTodos().iterator();
+        for (Object obj:getVisitTodos()) {
+            if (!otherPersonVisitTodos.hasNext() || !obj.equals(otherPersonVisitTodos.next())) {
+                return false;
+            }
+        }
```

Comment:
This code seems to be the same as in `EditCommand.EditPersonDescriptor`. Since checking `Collection<VisitTodo>` equality seems to be a much used function, perhaps it could be put in a utility class somewhere, if you feel that creating a dedicated class such as `VisitTodoCollection` with an `equals()` method is unnecessary.